aries_cloudagent/indy/models/cred_abstract.py

```diff
@@ -153,200 +152,3 @@ class Meta:
         required=True,
         metadata={"description": "Key correctness proof"},
     )
-
-
-class AnoncredsLinkSecret(BaseModel):
-    """Anoncreds Link Secret Model."""
-
-    class Meta:
-        """AnoncredsLinkSecret metadata."""
-
-        schema_class = "AnoncredsLinkSecretSchema"
-
-    def __init__(
-        self,
-        nonce: str = None,
-        cred_def_id: str = None,
-        key_correctness_proof: str = None,
-        **kwargs,
-    ):
-        """Initialize values for AnoncredsLinkSecret."""
-        super().__init__(**kwargs)
-        self.nonce = nonce
-        self.cred_def_id = cred_def_id
-        self.key_correctness_proof = key_correctness_proof
-
-
-class AnoncredsLinkSecretSchema(BaseModelSchema):
-    """Anoncreds Link Secret Schema."""
-
-    class Meta:
-        """AnoncredsLinkSecret schema metadata."""
-
-        model_class = AnoncredsLinkSecret
-        unknown = EXCLUDE
-
-    nonce = fields.Str(
-        required=True,
-        validate=NUM_STR_WHOLE_VALIDATE,
-        metadata={
-            "description": "Nonce in credential abstract",
-            "example": NUM_STR_WHOLE_EXAMPLE,
-        },
-    )
-
-    cred_def_id = fields.Str(
-        required=True,
-        validate=INDY_CRED_DEF_ID_VALIDATE,
-        metadata={
-            "description": "Credential definition identifier",
-            "example": INDY_CRED_DEF_ID_EXAMPLE,
-        },
-    )
-
-    key_correctness_proof = fields.Nested(
-        IndyKeyCorrectnessProofSchema(),
-        required=True,
-        metadata={"description": "Key correctness proof"},
-    )
-
-
-class DidcommSignedAttachment(BaseModel):
-    """Didcomm Signed Attachment Model."""
-
-    class Meta:
-        """DidcommSignedAttachment metadata."""
-
-        schema_class = "DidcommSignedAttachmentSchema"
-
-    def __init__(
-        self,
-        algs_supported: Sequence[str] = None,
-        did_methods_supported: Sequence[str] = None,
-        nonce: str = None,
-        **kwargs,
-    ):
-        """Initialize values for DidcommSignedAttachment."""
-        super().__init__(**kwargs)
-        self.algs_supported = algs_supported
-        self.did_methods_supported = did_methods_supported
-        self.nonce = nonce
-
-
-class DidcommSignedAttachmentSchema(BaseModelSchema):
-    """Didcomm Signed Attachment Schema."""
-
-    class Meta:
-        """Didcomm signed attachment schema metadata."""
-
-        model_class = DidcommSignedAttachment
-        unknown = EXCLUDE
-
-    algs_supported = fields.List(fields.Str(), required=True)
-
-    did_methods_supported = fields.List(fields.Str(), required=True)
-
-    nonce = fields.Str(
-        required=True,
-        validate=NUM_STR_WHOLE_VALIDATE,
-        metadata={
-            "description": "Nonce in credential abstract",
-            "example": NUM_STR_WHOLE_EXAMPLE,
-        },
-    )
-
-
-class BindingMethod(BaseModel):
```

Comment:
huh no one else uses this we are sure?

Reply:
Yes, it has just moved to a new location. And used only in vc_di.